win32ss/user/ntuser/display.c

```diff
@@ -151,43 +151,25 @@ InitDisplayDriver(
     return pGraphicsDevice;
 }
 
-NTSTATUS
-NTAPI
-InitVideo(VOID)
+static
+VOID
+GdiUpdateGraphicsDeviceList(VOID)
```

Comment:
I suggest to put this into gdi/eng/device.c

Reply:
Done